modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1752,6 +1752,10 @@ public String getSchemaType(Schema schema) {
      */
     @SuppressWarnings("static-method")
     public String toAllOfName(List<String> names, ComposedSchema composedSchema) {
+        Map<String, Object> exts = composedSchema.getExtensions();
+        if (exts != null && exts.containsKey("x-allOf-name")) {
+            return (String) exts.get("x-allOf-name");
```

Comment:
We want to standardize vendor extension naming with lower cases only separated by hyphens so  what about using`x-all-of-name` instead?